linux/LinuxProcess.c

```diff
@@ -421,6 +454,14 @@ long LinuxProcess_compare(const void* v1, const void* v2) {
    #endif
    case OOM:
       return (p2->oom - p1->oom);
+   #ifdef HAVE_DELAYACCT
```

Comment:
These comparisons are not working because they are float values. See the PERCENT_CPU comparison in Process.c.